test/lock-parser/fixtures/restriction-overwrite/out.json

```diff
@@ -0,0 +1,232 @@
+{
```

Comment:
same as before a list of available groups would be nice, and all restrictions. Based on the format of this file it seems like:
- we want to see each group
- each group may have a restriction
- each group may have multiple registries
- each registry may have multiple remotes

Reply:
Just at the top would be great, looks like most data is preserved which is great

Reply:
We now index the output by group which more closely matches our file contents and each group shows the information as requested. Showing this information at the root of the result object would not make sense as the data is group dependent.